python/tidy/servo_tidy/tidy.py

```diff
@@ -66,7 +66,7 @@
     "//dev.w3.org/fxtf",
     "//dvcs.w3.org/hg",
     "//dom.spec.whatwg.org",
-    "//domparsing.spec.whatwg.org",
+    "//w3c.github.io/DOM-Parsing",
```

Comment:
We can remove this line now; it's already covered by the w3c.github.io further down in the list.